tariff-logic/taric-footwear.test.js

```diff
@@ -15,674 +15,1000 @@ test.each([
+  ],
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'yes' },
+      { questionKey: 'heightOfSoleAndHeel', answerKey: 'no' },
+      { questionKey: 'lengthOfInsole', answerKey: 'no' }
+    ],
+    '6403993100'
+  ],
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'yes' },
+      { questionKey: 'heightOfSoleAndHeel', answerKey: 'no' },
+      { questionKey: 'lengthOfInsole', answerKey: 'yes' }
+    ],
+    { code: '640399', questionKey: 'genderType' }
+  ],
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'yes' },
+      { questionKey: 'heightOfSoleAndHeel', answerKey: 'no' },
+      { questionKey: 'lengthOfInsole', answerKey: 'yes' },
+      { questionKey: 'genderType', answerKey: 'men' }
+    ],
+    '6403993600'
+  ],
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'yes' },
+      { questionKey: 'heightOfSoleAndHeel', answerKey: 'no' },
+      { questionKey: 'lengthOfInsole', answerKey: 'yes' },
+      { questionKey: 'genderType', answerKey: 'women' }
+    ],
+    '6403993800'
+  ],
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'yes' },
+      { questionKey: 'heightOfSoleAndHeel', answerKey: 'no' },
+      { questionKey: 'lengthOfInsole', answerKey: 'yes' },
+      { questionKey: 'genderType', answerKey: 'unisex/other' }
+    ],
+    '6403993300'
+  ],
+  // Vamp = no
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' }
+    ],
+    { code: '640399', questionKey: 'slippers' }
+  ],
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' },
+      { questionKey: 'slippers', answerKey: 'yes' }
+    ],
+    '6403995000'
+  ],
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' },
+      { questionKey: 'slippers', answerKey: 'no' }
+    ],
+    { code: '6403xx', questionKey: 'winterSports' }
+  ],
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' },
+      { questionKey: 'slippers', answerKey: 'no' },
+      { questionKey: 'winterSports', answerKey: 'yes' }
+    ],
+    { code: '6403xx', questionKey: 'skiBoots' }
+  ],
+  // winterSports = yes
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' },
+      { questionKey: 'slippers', answerKey: 'no' },
+      { questionKey: 'winterSports', answerKey: 'yes' },
+      { questionKey: 'skiBoots', answerKey: 'skiBoots' }
```

Comment:
Lack test case if skiBoots => snowboardBoots? (that also ends up as 6403 12 00 00)